packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -0,0 +1,468 @@
+          label: channel.name,
+        }));
+      },
+      enabledChannelOptions() {
+        return this.channelOptionsList.filter(c => !c.disabled);
+      },
+      langId() {
+        return Object.keys(this.value.languages)[0];
+      },
+      accessId() {
+        return Object.keys(this.value.accessibility_labels)[0];
+      },
+      selectedLanguage() {
+        if (!this.langId && this.enabledLanguageOptions.length === 1) {
+          return this.enabledLanguageOptions[0];
+        }
+        return this.languageOptionsList.find(o => o.value === this.langId) || {};
+      },
+      selectedAccessibilityFilter() {
+        if (!this.accessId && this.enabledAccessibilityOptions.length === 1) {
+          return this.enabledAccessibilityOptions[0];
+        }
+        return this.accessibilityOptionsList.find(o => o.value === this.accessId) || {};
+      },
+      levelId() {
+        return Object.keys(this.value.grade_levels)[0];
+      },
+      selectedLevel() {
+        if (!this.levelId && this.enabledContentLevels.length === 1) {
+          return this.enabledContentLevels[0];
+        }
+        return this.contentLevelsList.find(o => o.value === this.levelId) || {};
+      },
+      channelId() {
+        return Object.keys(this.value.channels)[0];
+      },
+      selectedChannel() {
+        if (!this.channelId && this.enabledChannelOptions.length === 1) {
+          return this.enabledChannelOptions[0];
+        }
+        return this.channelOptionsList.find(o => o.value === this.channelId) || {};
+      },
+    },
+    methods: {
+      isChecked(inputKey, value) {
+        return this.isSelected(inputKey, value) || this.isEnabledButNotSelected(inputKey, value);
+      },
+      isSelected(inputKey, value) {
+        return this.value[inputKey][value.value] === true;
+      },
+      isEnabledButNotSelected(inputKey, value) {
+        return (
+          !this.isSelected(inputKey, value) &&
+          {
+            languages: this.enabledLanguageOptions,
+            channels: this.enabledChannelOptions,
+            accessibility_labels: this.enabledAccessibilityOptions,
+            grade_levels: this.enabledContentLevels,
+          }[inputKey].includes(value)
+        );
+      },
+      accordionHeaderStyles(selected) {
+        return {
+          padding: `0.25em 0 0.25em ${selected ? '0.5em' : '0.75em'}`,
+          background: selected ? this.selectedHighlightColor : this.$themePalette.grey.v_100,
+          borderLeft: selected ? `0.25em solid ${this.$themeTokens.primary}` : 'none',
+        };
+      },
+      handleChange(field, value) {
+        const prevFieldValue = this.value[field];
+        if (value && this.isSelected(field, value)) {
+          delete prevFieldValue[value.value];
+          this.$emit('input', { ...this.value, [field]: prevFieldValue });
+        } else {
+          this.$emit('input', {
+            ...this.value,
+            [field]: { ...prevFieldValue, [value.value]: true },
```

Comment:
With this, the channels are working as a checkbox rather than as a radio button, and we can select both of them but just one appears as selected

![image](https://github.com/user-attachments/assets/b2644142-658d-48ce-a1b2-926a40822a7c)


I wonder if we should change the radio buttons to be checkboxes or change this `handleChange` method to support the radio button behaviour to have just one selected value and remove the rest.

Reply:
Thank you Alex, I've resolved this issue